dataproc/create_cluster.py

```diff
@@ -58,4 +64,15 @@ def create_cluster(project_id, region, cluster_name):
 
     # Output a success message.
     print('Cluster created successfully: {}'.format(result.cluster_name))
-    # [END dataproc_create_cluster]
+
+
+if __name__ == "__main__":
+    if len(sys.arv) != 4:
+        print("INSUFFICIENT ARGS: Please provide a "
```

Comment:
How about:

```python
if len(sys.argv) < 4:
    sys.exit('python create_cluster.py project_id region cluster_name')
```
?

Reply:
Fixed.

Reply:
The messages seems reverted to the previous state?

Reply:
Fixed again.